src/commands/dev/dev.ts

```diff
@@ -155,6 +157,16 @@ export const dev = async (options: OptionValues, command: BaseCommand) => {
     siteInfo,
   })
 
+  if (!options.offline && !options.offlineEnv) {
+    await setupAIGateway({ api, env, siteID: site.id, siteURL: siteUrl })
+  }
+
+  // eslint-disable-next-line @typescript-eslint/no-unnecessary-condition
+  if (env.AI_GATEWAY?.value) {
+    process.env.AI_GATEWAY = env.AI_GATEWAY.value
+    log(`${NETLIFYDEVLOG} AI Gateway configured for AI provider SDK interception`)
```

Comment:
I wonder if this log message is going to be useful to people. 🤔

Reply:
🤔 Hmm I would expect some type of log, but open to suggestions on wording